crates/hdk/README.md

```diff
@@ -219,15 +193,15 @@ Holochain exposes a list of holochain-specific things that a wasm can do.
 For detailed documentation of the full list, see the `core/ribosome` module
 inside core, but some illustrative examples include:
 
-- `emit_signal`: publish data to subscribed clients
-- `encrypt` & `decrypt`: use the agent's keypair to encrypt and decrypt data
+- `emit_signal`: publish data to subscribed interface clients
 - `sign`: use the agent's keypair to sign some data
-- `commit_entry`: save some data to the local source chain and broadcast it to
+- `create_entry`: save some data to the local source chain and broadcast it to
   the DHT to be redundantly validated and stored
 - `get_entry`: retrieve some data from local or the network given its hash
 - `create_link`: create graph style relationships (links) between entries
 - `get_links`: retrive links between entries using the DHT as a graph database
-- `send`: send data directly to a known peer on the network
+- `remote_signal`: send data directly to known peers on the network without waiting for a response
+- `remote_call`: perform a remote procedure call on a peers node, if you're authorized
```

Comment:
This one should be `call_remote`, which maybe indicates that `remote_signal` should be renamed to `emit_signal_remote` to make it similar to both `call_remote` and `emit_signal` in the next HDK sweep?

Reply:
I would agree with this :D

Reply:
yeah that'd be a nice change. I will update the call_remote

Reply:
can you capture that notion somewhere else @pdaoust and I'll resolve this conversation

Reply:
Happy to; I'll create a PR in fact.